src/globus_cli/types.py

```diff
@@ -46,3 +51,8 @@
 JsonValue: TypeAlias = t.Union[
     int, float, str, bool, None, t.List["JsonValue"], t.Dict[str, "JsonValue"]
 ]
+
+
+ServiceNameLiteral: TypeAlias = Literal[
+    "auth", "transfer", "groups", "search", "timer", "flows"
+]
```

Comment:
It'd be nice to somehow link this to `_CLIScopeRequirements`. If not programatically at least a comment saying "hey if you add a service make sure to update that list over there!"

I know you can't dynamically generate a literal & expect type checking semantics but effectively (as far as I understand) this is just a static list of the keys in that object.

Reply:
Let me add a test for this. We just need to do the `typing.get_args` magic and compare against the keys.
It doesn't even need to work in all python versions, as long as it works in 3.11+